### PR TITLE
Minor design changes to remove "JSQ" branding :)

### PR DESCRIPTION
**Pull request by jessesquires**
Hey @vermont42 ! 👋 

Really happy to see that you were able to build your own site by forking mine. 😄 

I wanted to send you this PR to remove some of my "branding" (colors, fonts, etc.). Everything that I removed now just falls back to Bootstrap defaults (which are great). 

- Remove custom UbuntuMono font
- Remove custom link and button colors

I'm hoping you'll accept these changes!

Other notes:

- I noticed you have `_site/` checked in. Usually you want this to be ignored. Was that intentional?

### CODE REVIEW

**Comment by jessesquires** [comment]
A few screenshots of what this looks like now:

screen shot 2018-04-15 at 7 00 31 pm

screen shot 2018-04-15 at 7 13 10 pm

File: _site/feed.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <subtitle>A Blog by Josh Adams</subtitle>
     <link href="http://racecondition.software/feed.xml" rel="self" />
     <link href="http://racecondition.software" />
-    <updated>2018-04-08T16:56:34-07:00</updated>
+    <updated>2018-04-15T19:03:07-07:00</updated>
```
Comment (vermont42) [inline _site/feed.xml:7]: Why this change?

Comment (jessesquires) [reply _site/feed.xml:7]: Ah, this is because you check-in the `_site/` directory. So this is an artifact of running `jekyll build`.

From the original PR description: 

> I noticed you have `_site/` checked in. Usually you want this to be ignored. Was that intentional?

You probably want `_site/` to be in your `.gitignore`. But, it might depend on your hosting setup.

Comment (jessesquires) [reply _site/feed.xml:7]: Oh, I just realized you don't have a `.gitignore` 😊 

I'd recommend adding one. 😄

Comment (vermont42) [reply _site/feed.xml:7]: I will do so.

**Comment by jessesquires** [comment]
Thanks @vermont42 ! 👍
